source/Core/Threads/UI/logic/Soldering.cpp

```diff
@@ -39,7 +39,7 @@ OperatingMode handleSolderingButtons(const ButtonState buttons, guiContext *cxt)
         cxt->scratch_state.state2 = 1;
         break;
       }
-    /*Fall through*/
+    /*Fall throuhg*/
```

Comment:
Erm? Why the change?

Reply:
```suggestion
    /*Fall through*/
```
Likely was a mistake while scrolling thru where I highlighted and dragged.